Resources/views/Block/block_stats_base.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+{% extends sonata_block.templates.block_base %}
+
+{% block block %}
```

Comment:
`block` is a reserved keyword, this could be confusing. Consider other names, like `outer_block`, `inner_block`.